packages/react-components/react-breadcrumb/src/components/BreadcrumbLink/useBreadcrumbLink.ts

```diff
@@ -30,6 +30,7 @@ export const useBreadcrumbLink_unstable = (
     root: getNativeElementProps('a', {
       'aria-current': current ? props['aria-current'] ?? 'page' : undefined,
       ref,
+      tabIndex: focusMode === 'off' ? -1 : undefined,
```

Comment:
does it make sense for breadcrumb links to not be focusable? they should not be rendered semantically as a `a` tag then

Reply:
We discussed with Sarah that it won't hurt to turn it off. But we talked about non-interactive controls and in our case if people expect non-interactive Breadcrumb they can use only BreacrumbItem. I'll remove `off` focus mode

Reply:
resolving as this is fixed